src/test/java/gale/GaleTest.java

```diff
@@ -0,0 +1,42 @@
+package gale;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class GaleTest {
+    private Gale gale;
+    private ByteArrayOutputStream outputCapture;
+
+    @BeforeEach
+    public void setUp() {
+        outputCapture = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(outputCapture));
+        gale = new Gale("src/main/java/data/galeTest.txt");
+    }
+
+    public void simulateUserInput(String input) {
+        System.setIn(new ByteArrayInputStream(input.getBytes()));
+    }
+
+    @Test
+    public void testAddTask_success() throws GaleException {
```

Comment:
Unfortunately again, this doesn't follow test case naming standards!
> Underscores may be used in test method names using the following three part format featureUnderTest_testScenario_expectedBehavior()

> e.g. sortList_emptyList_exceptionThrown() getMember_memberNotFound_nullReturned

More details [here!](https://se-education.org/guides/conventions/java/intermediate.html#naming)

Reply:
For this, it could be addToDoTask_validInput_successMessage()?

Reply:
same review; 

doesn't follow test case naming standards:(
for help [click here](https://se-education.org/guides/conventions/java/intermediate.html#naming)

the correct format could be addToDoTask_validInput_successMessage() (maybe)